Modules/BookingManager/classes/class.ilObjBookingPool.php

```diff
@@ -200,7 +200,7 @@ public function delete() : bool
         return true;
     }
     
-    public function cloneObject($a_target_id, $a_copy_id = 0, $a_omit_tree = false)
+    public function cloneObject(int $a_target_id, int $a_copy_id = 0, $a_omit_tree = false) : ?ilObject
```

Comment:
...no datatype 'bool' for '$a_omit_tree' yet

Reply:
Hello @utesche , that's probably lost in the amount of files. Thank you very much for your hint. I'll fix that in an extra PR. Greetings Daniel.

Reply:
You will find the PR with the fix here: https://github.com/ILIAS-eLearning/ILIAS/pull/4113